src/app/dataset_storage.py

```diff
@@ -1,9 +1,18 @@
 import json
+import os
 
 from google.cloud import storage
 
 storage_client = storage.Client()
 
+# If the local development is used then, it will use those values to create the local storage.
+if os.environ.get("DATASET_BUCKET_NAME"):
```

Comment:
You're calling `os.environ.get("DATASET_BUCKET_NAME")` three times. Can you do this once and put into a variable.

Reply:
Sorted.